cmd/helm-sync/main.go

```diff
@@ -127,7 +127,7 @@ func main() {
 			Namespace:               *flNamespace,
 			DeployNamespace:         *flDeployNamespace,
 			Values:                  *flValues,
-			ValuesFileSources:       strings.Split(*flValuesFileSources, ","),
+			ValuesFileRefs:          strings.Split(*flValuesFileRefs, ","),
```

Comment:
"ValuesFilePaths" is a better name in this context. the object has already been exported to file, so it's not a ref any more. Same with the flag var name.

Reply:
Done